paasta_tools/contrib/get_running_task_allocation.py

```diff
@@ -281,7 +281,7 @@ def parse_args() -> argparse.Namespace:
         default="paasta",
     )
     parser.add_argument(
-        "additional_namespaces",
+        "--additional_namespaces",
```

Comment:
nit: i think i doubly messed up here and also forgot to make this dash-separated
```suggestion
        "--additional-namespaces",
```